harfbuzz/VITABUILD

```diff
@@ -1,21 +1,18 @@
 pkgname=harfbuzz
-pkgver=1.4.8
+pkgver=2.8.1
 pkgrel=1
-url="http://harfbuzz.org/"
-source=("https://github.com/behdad/harfbuzz/releases/download/${pkgver}/harfbuzz-${pkgver}.tar.bz2")
-sha256sums=('ccec4930ff0bb2d0c40aee203075447954b64a8c2695202413cc5e428c907131')
-prepare() {
-  cd $pkgname-$pkgver
-}
+url="https://github.com/harfbuzz/harfbuzz"
+source=("https://github.com/harfbuzz/harfbuzz/releases/download/$pkgver/$pkgname-$pkgver.tar.xz")
+sha256sums=('4124f663ec4bf4e294d9cf230668370b4249a48ff34deaf0f06e8fc82d891300')
 
 build() {
-  cd harfbuzz-$pkgver
+  cd $pkgname-$pkgver
   mkdir build && cd build
   cmake .. $CMARGS -DCMAKE_INSTALL_PREFIX=$prefix
```

Comment:
There are few issues with harfbuzz:
1. Now that icu4c is merged, we can enable icu support in harfbuzz
2. Building with cmake doesn't install pkgconfig config (and, libass, for example, requires that, although, that can be worked around), only automake and meson builds are.

Reply:
I removed harfbuzz from this PR